### PR TITLE
Edited Binary Studio experience

### PR DESCRIPTION
**Pull request by dasha-korotkykh**
Updated wording for Academy, trips, office administration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
+                            <li>plan and implement Marketing campaigns for Binary Studio Academy</li>
+                            <li>rebrand Academy website, social media channels, enhance brand-awareness through radio-podcast and articles</li>
                           </ul>
                         </div>
                       </li>
                       <li><strong>social media management</strong> &mdash; write internal weekly newsletter <strong>Pirozhki</strong> &#x1f96e;, transmit Binary Studio experiences, increase brand awareness, and keep building online reputation through SMM channels (Facebook, Twitter, Instagram, etc.)</li>
                       <li>help select, order, prepare and distribute <strong>presents for employees</strong> &#x1f381;&#x1f384;&#x1f385;</li>
-                      <li>help with any kind of <strong>administrative tasks</strong>, e.g. making sure that our office is always an inspiring work environment, etc.</li>
+                      <li>help with any kind of <strong>administrative tasks</strong>, making sure that our office is always clean, inspiring and comfortable work environment, check supplies and coordinate cleaning staff</li>
```
Comment (volodymyr-kushnir) [inline index.html]: Oxford comma before "and comfortable work environment" (but not before "and coordinate cleaning stuff").

File: index.html [hunk 2]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
                         <div>
                           <ul>
-                            <li>Establish partnerships with local Universities</li>
-                            <li>Сooperate with Universities by sponsorships, job fairs, open lectures etc.</li>
-                            <li>Organize and hold events for students </li>
-                            <li>Speak at students conferences promoting Binary Studio and Binary Studio Academy</li>
-                            <li>Plan and implement Marketing campaigns for Binary Studio Academy</li>
-                            <li>Rebrand the Binary Studio Academy website, crystallize positioning</li>
-                            <li>make Binary Studio Academy known within university students, prepare curriculum with developers-coaches, select best 10% of candidates (including about 200 interviews a year), moderate product meetings and make sure things generally don't go south</li>
+                            <li>prepare curriculum with developers-coaches, select best 10% of candidates (including about 200 interviews a year), moderate product meetings and make sure things generally don't go south</li>
```
Comment (volodymyr-kushnir) [inline index.html]: "prepare curriculum with coaches, select best 10% of candidates (by making about 200 interviews), moderate product meetings, and make sure things generally don't go south"

File: index.html [hunk 3]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
+                            <li>establish partnerships with top-8 Universities, cooperate with them by sponsorships, job fairs, open lectures</li>
```
Comment (volodymyr-kushnir) [inline index.html]: "establish partnerships with top 8 universities in Ukraine, cooperate with them by sponsorships, job fairs, open lectures".

File: index.html [hunk 4]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
+                            <li>organize and hold events for students </li>
```
Comment (volodymyr-kushnir) [inline index.html]: Remove trailing space.

File: index.html [hunk 5]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
+                            <li>plan and implement Marketing campaigns for Binary Studio Academy</li>
```
Comment (volodymyr-kushnir) [inline index.html]: Maybe "marketing campaigns", lowercase?

File: index.html [hunk 6]
```diff
@@ -652,19 +651,17 @@ <h6>Recent</h6>
...
+                            <li>rebrand Academy website, social media channels, enhance brand-awareness through radio-podcast and articles</li>
```
Comment (volodymyr-kushnir) [inline index.html]: "brand awareness", "radio podcast".

File: index.html [hunk 7]
```diff
@@ -651,17 +651,17 @@ <h6>Recent</h6>
                         <label for="binary-studio-academy" class="summary"><strong>coordinate <a href="https://academy.binary-studio.com/" rel="external">Binary Studio Academy</a> &#x1f393;</strong></label>
                         <div>
                           <ul>
-                            <li>prepare curriculum with developers-coaches, select best 10% of candidates (including about 200 interviews a year), moderate product meetings and make sure things generally don't go south</li>
-                            <li>establish partnerships with top-8 Universities, cooperate with them by sponsorships, job fairs, open lectures</li>
-                            <li>organize and hold events for students </li>
-                            <li>plan and implement Marketing campaigns for Binary Studio Academy</li>
-                            <li>rebrand Academy website, social media channels, enhance brand-awareness through radio-podcast and articles</li>
+                            <li>repare curriculum with coaches, select best 10% of candidates (by making about 200 interviews), moderate product meetings, and make sure things generally don't go south</li>
```
Comment (volodymyr-kushnir) [inline index.html]: You lost "p" — "prepare" curriculum.

File: index.html [hunk 8]
```diff
@@ -651,17 +651,17 @@ <h6>Recent</h6>
...
+                            <li>establish partnerships with top-8 universities in Ukraine, cooperate with them by sponsorships, job fairs, open lectures</li>
```
Comment (volodymyr-kushnir) [inline index.html]: Nobody writes "top-8". Replace with "top 8".

File: index.html (hunk 7 above)
Comment (dasha-korotkykh) [reply index.html]: right. looks more like repairing activity to me every time

**Pull request merged**